keyboards/reviung/reviung46/keymaps/via/keymap.c

```diff
@@ -0,0 +1,51 @@
+// Copyright 2021 gtips (@gtips)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+	_BASE,
+	_LOWER,
+	_RAISE,
+	_ADJUST
+};
+
+#define LOWER  MO(_LOWER)
+#define RAISE  MO(_RAISE)
+#define ADJUST MO(_ADJUST)
+
```

Comment:
As above.
```suggestion
```

Reply:
Thanks.
As above.
If I remove lines 14-15, lines 19-20 and 49-51 don't work.

Reply:
See above.